### PR TITLE
feat(upgrade-job): add version info log

### PR DESCRIPTION
**Pull request by niladrih**
Adds version log when initializing.

### CODE REVIEW

**Comment by niladrih** [comment]
bors try

**Review by sinhaashish — APPROVED** [review]

File: k8s/upgrade/src/bin/upgrade-job/main.rs [hunk 1]
```diff
@@ -23,6 +24,11 @@ mod upgrade;
 #[tokio::main]
 async fn main() -> Result<()> {
     init_logging();
+    info!(
```
Comment (tiagolobocastro) [inline k8s/upgrade/src/bin/upgrade-job/main.rs:27]: I think there's a mcro for this already: utils::print_package_info!();

Comment (niladrih) [reply k8s/upgrade/src/bin/upgrade-job/main.rs:27]: Let me try that one out..

**Comment by niladrih** [comment]
bors try

File: k8s/upgrade/src/bin/upgrade-job/main.rs (hunk 1 above)
Comment (niladrih) [reply k8s/upgrade/src/bin/upgrade-job/main.rs]: Alright using print_package_info!.

**Review by tiagolobocastro — APPROVED** [review]

**Comment by tiagolobocastro** [comment]
Btw once we have cli args this can also be useful? Example:
```rust
println!("Using options: {:?}", &cli_args);
```

**Comment by niladrih** [comment]
bors merge

**Comment by niladrih** [comment]
> Btw once we have cli args this can also be useful? Example:
> 
> ```rust
> println!("Using options: {:?}", &cli_args);
> ```

That's a good idea..

**Pull request merged**